packages/common/src/intl/context/IntlContext.tsx

```diff
@@ -10,12 +10,15 @@ import { PropsWithChildrenOnly } from '@common/types';
 const defaultNS = 'common';
 export const IntlProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
   React.useEffect(() => {
+    if (i18next.isInitialized) return;
```

Comment:
this isn't necessary now - but left it in. I had an issue because I was running `init` in the testing provider which then screwed up the site 🙄